source/Generic/GameRelations/GameRelationsSettings.cs

```diff
@@ -185,6 +189,27 @@ private void RemoveSelectedItemsFromExclude(ObservableCollection<DatabaseObject>
             }
         }
 
+        private string GetResourceString(string key) => plugin.PlayniteApi.Resources.GetString(key);
+
+        public void SetAdvancedSectionDefaults()
+        {
+            Settings.SimilarGamesControlSettings.JacardSimilarityPerField = 0.73D;
+            Settings.SimilarGamesControlSettings.FieldSettings.Clear();
+            settings.SimilarGamesControlSettings.FieldSettings.Add(new SimilarGamesFieldSettings(GameField.TagIds, GetResourceString("LOCTagsLabel"), true, 1));
+            settings.SimilarGamesControlSettings.FieldSettings.Add(new SimilarGamesFieldSettings(GameField.GenreIds, GetResourceString("LOCGenresLabel"), true, 1.2));
+            settings.SimilarGamesControlSettings.FieldSettings.Add(new SimilarGamesFieldSettings(GameField.CategoryIds, GetResourceString("LOCCategoriesLabel"), true, 1.3));
```

Comment:
I don't think the names should be set during their creation. This creates the issue that if language or source string changes, they'll stay the previous incorrect name so it would be better to resolve at runtime.